### DISCUSSION TITLE
Start Menu Fullscreen - Edge Installed Application not displaying

### DISCUSSION THREAD

**Discussion by sshaikKxt** [valinet/ExplorerPatcher#2426] [Q&A] [answered]
Hi there, I am using Win11, 22H2, 22621.2506.
I am happy for this ExplorerPatcher solution its such a lifesaver couldnt do without the Full screen Start Menu.
I installed numerous Microsoft Edge Apps on my previous laptop running on Win10 and pin them as tiles on the Start menu.
They reflect onto the Start Menu for the Win11, but not on the ExplorerPatcher Win10.
Did i miss something here or is it not enabled?

![image](https://github.com/valinet/ExplorerPatcher/assets/111260343/e2c119f4-75a1-4a54-bc98-ed50e218bc3b)

**Comment by pyrates999** [comment]
Please post explorer patcher version.

**Comment by sshaikKxt** [comment]
V 22000.613.44.1

![image](https://github.com/valinet/ExplorerPatcher/assets/111260343/e5081f56-9b19-4f68-b0e5-ae4b180cf6b5)

**Comment by sshaikKxt** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/111260343/d3f09553-b71d-43aa-98f8-61cdb3c8cc27)

![image](https://github.com/valinet/ExplorerPatcher/assets/111260343/3a792e85-d886-437a-8e61-2e973a75697b)

This is how i would normally 'install' apps from website using Microsoft Edge websites which would act as an 'app' in a single window. Previously on my Win10 I would 'install' frequently used websites as an application and tile them onto the Full Screen Start Menu.

Also, i noticed that Apps installed through the Windows Store does not reflect onto the explorerPatch Start Menu.

Thank you.

**Comment by pyrates999** [comment, marked as answer]
That's an old EP version.  Install the latest from here: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2428.59.1_a7c87ce

**Reply by sshaikKxt** [reply]
it works!

**Reply by sshaikKxt** [reply]
Thank you!

**Reply by pyrates999** [reply]
Always make sure your version of EP is up to date.
